packages/mds-geography/logger.ts

```diff
@@ -0,0 +1,3 @@
+import logger from '@mds-core/mds-logger'
+
+export const GeographyLogger = logger.createLogger('mds-geography')
```

Comment:
I wish we could rename some of these packages to include `-api` where appropriate. This really should be GeographyApiLogger, IMO.